docs/userguide.rst

```diff
@@ -251,13 +253,19 @@ The above example configures 'somename.eth' to resolve to the address of your pr
 Transferring a name
 -------------------
 
-You can transfer ownership of a name you own in the ENS registry to someone else using `setOwner`:
+You can transfer ownership of a name you own in the ENS registry to another trusted account using `setOwner`:
```

Comment:
This change seems superfluous. You can transfer ownership to any account, whether you trust them or not.